src/main/java/practice/StreamPractice.java

```diff
@@ -23,7 +35,12 @@ public int findMinEvenNumber(List<String> numbers) {
      * But before that subtract 1 from each element on an odd position (having the odd index).
      */
     public Double getOddNumsAverage(List<Integer> numbers) {
-        return 0D;
+        return IntStream.range(0, numbers.size())
+                .map(n -> n % 2 != 0 ? numbers.get(n) - 1 : numbers.get(n))
```

Comment:
```suggestion
                .map(i -> i % 2 != 0 ? numbers.get(i) - 1 : numbers.get(i))
```
I think it's better.
i - index

Reply:
fixed